src/container/cli.c

```diff
@@ -332,94 +332,48 @@ dc_cont_destroy(tse_task_t *task)
 	crt_endpoint_t		 ep;
 	crt_rpc_t		*rpc;
 	struct cont_args	 arg;
+	uuid_t			 uuid;
+	const char		*label;
 	int			 rc;
 
 	args = dc_task_get_args(task);
 
-	if (uuid_is_null(args->uuid))
+	if (daos_uuid_valid(args->uuid)) {
+		/** Backward compatibility, we are provided a UUID */
+		label = NULL;
+		uuid_copy(uuid, args->uuid);
+	} if (daos_label_is_valid(args->cont)) {
```

Comment:
(style)  trailing statements should be on next line (or did you mean 'else if'?)